staging/src/k8s.io/api/networking/v1/types.go

```diff
@@ -305,13 +305,13 @@ type IngressSpec struct {
 	// IngressClassName is the name of an IngressClass cluster resource. Ingress
 	// controller implementations use this field to know whether they should be
 	// serving this Ingress resource, by a transitive connection
-	// (controller -> IngressClass -> Ingress resource). Although the
+	// (controller → IngressClass → Ingress resource). Although the
```

Comment:
I know it's allowed, but I'd prefer to keep the comments ASCII if we can

Reply:
@sftim requested to use those, but I don't really mind to change it again ^^

Reply:
It'd be good to codify this sort of thing in review guidelines and perhaps even CI tests.

For the [rendered API docs](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/), arrows are better than ASCII art. In source code, maybe we're worried about compatibility with older OSs?

Reply:
@sftim, @thockin I understand that this topic is something that should be clarified long term. What can I do to get this PR merged?

Reply:
For the arrows here, I'm fine with either approach. I hadn't spotted the ASCII-comments-only restriction.